go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/client9/misspell v0.3.4
 	github.com/go-jose/go-jose/v3 v3.0.3
 	github.com/golangci/golangci-lint v1.59.1
-	github.com/google/go-github/v63 v63.0.0
+	github.com/google/go-github/v64 v64.0.0
```

Comment:
we need to bump this to v65 and remove the indirect below, running `go mod tidy` should then also remove v64 from `go.sum`

Reply:
https://github.com/integrations/terraform-provider-github/pull/2380/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6

Reply:
Sorry I was rushing and missed this step, done now

Reply:
thanks!